Documentation/platforms/xtensa/esp32s3/boards/esp32s3-eye/index.rst

```diff
@@ -0,0 +1,94 @@
+=============
+ESP32-S3-EYE
+=============
+
+The `ESP32-S3-EYE <https://github.com/espressif/esp-who/blob/master/docs/en/get-started/ESP32-S3-EYE_Getting_Started_Guide.md>`_ is a small-sized AI development board from Espressif featuring the ESP32-S3 CPU with a 2-Megapixel camera, an LCD display, a microphone, an 8 MB Octal PSRAM and an 8 MB flash.
+
+.. list-table::
+   :align: center
+
+   * - .. figure:: esp32s3_eye.png
+          :align: center
+
+Features
+========
+
+  - ESP32-S3 WROOM-1 Module
+  - USB micro USB port (it uses internal USB JTAG SERIAL device for flashing)
+  - Power LED
+  - LCD Display
+  - 2-MP Camera
+  - MEMS Microphone
+  - 8MB Octal PSRAM
+  - 8MB SPI Flash
+  - RST and BOOT buttons (BOOT accessible to user)
+
+Configurations
+==============
+
+nsh
+---
+
+Basic NuttShell configuration (console enabled in USB JTAG SERIAL Device, exposed via
+USB connection at 9600 bps).
+
+You can configure this board profile this way:
+
+  $ cd nuttxspace/nuttx
+  $ ./tools/configure.sh esp32s3-eye:nsh
+
+Flashing
+========
+
+Because ESP32-S3 doesn't use an external USB/Serial chip like others ESP32
+boards you should put it in programming mode this way:
+
+  1) Press and hold BOOT and RESET (RST) buttons at same time;
+  2) Release the RESET button and keep BOOT button pressed;
+  3) After one or more seconds release the BOOT button;
+  4) Run the flashing command: make flash ESPTOOL_PORT=/dev/ttyACM0
+
+Serial Console
+==============
+
+The internal USB JTAG SERIAL Device, by default, is used as serial console.
+It is normally detected by Linux host as a USB CDC/ACM serial device.
+
+It will show up as /dev/ttyACM[n] where [n] will probably be 0 (is it 1
```

Comment:
Documentation should not be written in first person.
Consider rewriting this sentence about personal use case or simply remove it.